qemu/tests/win_virtio_driver_update_by_installer.py

```diff
@@ -92,6 +92,8 @@ def change_virtio_media(cdrom_virtio):
                                                  gagent_install_cmd,
                                                  gagent_pkg_info_cmd)
 
+    win_driver_installer_test.viofs_svc_run(test, params, session)
```

Comment:
Suggest to update the test steps too (line16 - line 25), for we changed the test steps